webapps/src/main/java/org/camunda/bpm/webapp/impl/engine/ProcessEnginesFilter.java

```diff
@@ -289,13 +292,15 @@ protected String replacePlaceholder(String data,
                                       String appName,
                                       String engineName,
                                       String applicationPath,
-                                      String contextPath) {
+                                      String contextPath,
+                                      String cspNonce) {
     return data.replace(APP_ROOT_PLACEHOLDER, contextPath + applicationPath)
                .replace(BASE_PLACEHOLDER, String.format("%s%s/app/%s/%s/", contextPath,
                  applicationPath, appName, engineName))
                .replace(PLUGIN_PACKAGES_PLACEHOLDER, createPluginPackagesStr(appName,
                  applicationPath, contextPath))
-               .replace(PLUGIN_DEPENDENCIES_PLACEHOLDER, createPluginDependenciesStr(appName));
+               .replace(PLUGIN_DEPENDENCIES_PLACEHOLDER, createPluginDependenciesStr(appName))
+               .replace(CSP_NONCE_PLACEHOLDER, cspNonce);
```

Comment:
The questions is...
Should we in this case remove the whole nonce attribute from the `index.html`?
If we just replace the nonce placeholder with an empty string, then in the script tags there will be still left a  `nonce=""` attribute. I'm not sure if it would do anything but it's not nice.
I'll try how it behaves.

Reply:
If the only "impact" is not-so-nice HTML, I could live with it. Maybe you can test if it functions in different browsers? Alternatively, would it be possible to use the placeholder in the process engines filter to replace the entire attribute `nonce="abc"`?